.github/workflows/test.yaml

```
@@ -137,18 +137,36 @@ jobs:
           - python-version: '3.11'
 
           # Pre-release Python 3.13 with default settings.
-          # For now, no dependencies as cffi doesn't work on 3.13.
           - python-version: '3.13-dev'
-            tox-env: 'nodeps-withcov-posix'
-            job-name: 'nodeps-3.13'
-            # FIXME:https://github.com/twisted/twisted/issues/12060
-            # Also add twisted.python
-            # FIXME:https://github.com/twisted/twisted/issues/12061
-            # Add full twisted.test
-            trial-target: 'twisted.conch twisted.cred twisted.internet twisted.logger twisted.mail twisted.names twisted.pair twisted.persisted twisted.protocols twisted.runner twisted.spread twisted.test.test_defer twisted.trial twisted.web twisted.words'
-            # Disable concurrency for now, since we get occasional segfaults and
-            # this helps debug them:
-            trial-args: '--reactor=default'
+            tox-env: 'alldeps-withcov-posix'
+            job-name: 'alldeps-3.13'
+
+          # Pre-release Python 3.13 on macOS with default settings.
+          - python-version: '3.13-dev'
+            runs-on: 'macos-12'
+            tox-env: 'macos-withcov-alldeps'
+            job-name: 'macos-alldeps-3.13'
+
+          # pywin32 doesn't have wheels for Python 3.13 yet.
```

I think that we can keep this commented code.


But maybe move it after the other Window jobs.

It should make it easier to enable it later while we develop 3.13 support for Windows

Maybe add a link to the info about when the next pywin32 release will be available 

```suggestion
          # See https://github.com/mhammond/pywin32/issues/2294
          # pywin32 doesn't have wheels for Python 3.13 yet.
```